packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -90,11 +90,11 @@ export default {
     }
   },
   methods: {
-    toggle() {
-      this.iHaveChildrenActive = !this.iHaveChildrenActive;
+    toggleExpanded() {
+      this.sublinksExpanded = !this.sublinksExpanded
     },
     setData: function() {
-      this.iHaveChildrenActive = Boolean(this.link.imActive);
+      this.sublinksExpanded = this.link.iHaveChildrenActive || false;
```

Comment:
can we do something like this? `this.sublinksExpanded = !!this.link.iHaveChildrenActive;`

Reply:
We can, but I feel uncomfortable about double-negative logic operators - it's not comfortable to read, but just IMHO. If it's OK for everybody, let's remake it as you suggested.

Reply:
Left it this way, a lot more understandable then other approaches `Boolean(this.link.iHaveChildrenActive);`